pandas-stubs/core/groupby/generic.pyi

```diff
@@ -1,150 +1,191 @@
-    def median(self, **kwargs) -> Series[S1]: ...
-    def min(self, **kwargs) -> Series[S1]: ...
-    def nlargest(self, n: int = ..., keep: str = ...) -> Series[S1]: ...
-    def nsmallest(self, n: int = ..., keep: str = ...) -> Series[S1]: ...
-    def nth(self, n: int | Sequence[int], dropna: str | None = ...) -> Series[S1]: ...
-    def sum(
+        value: object | ArrayLike | None = None,
+        method: FillnaOptions | None = None,
+        axis: Axis | None | NoDefault = ...,
+        inplace: bool = False,
+        limit: int | None = None,
+        downcast: dict | None | NoDefault = ...,
+    ) -> Series[S1] | None: ...
+    def take(
         self,
-        numeric_only: bool = ...,
-        min_count: int = ...,
-        engine=...,
-        engine_kwargs=...,
+        indices: TakeIndexer,
+        axis: Axis | NoDefault = ...,
+        **kwargs,
     ) -> Series[S1]: ...
-    def prod(self, numeric_only: bool = ..., min_count: int = ...) -> Series[S1]: ...
-    def sem(self, ddof: int = ..., numeric_only: bool = ...) -> Series[float]: ...
-    def std(self, ddof: int = ..., numeric_only: bool = ...) -> Series[float]: ...
-    def var(self, ddof: int = ..., numeric_only: bool = ...) -> Series[float]: ...
-    def tail(self, n: int = ...) -> Series[S1]: ...
-    def unique(self) -> Series: ...
+    def skew(
+        self,
+        axis: Axis | NoDefault = ...,
+        skipna: bool = True,
+        numeric_only: bool = False,
+        **kwargs,
+    ) -> Series: ...
+    @property
+    def plot(self) -> GroupByPlot[Self]: ...
+    def nlargest(
+        self, n: int = 5, keep: Literal["first", "last", "all"] = "first"
+    ) -> Series[S1]: ...
+    def nsmallest(
+        self, n: int = 5, keep: Literal["first", "last", "all"] = "first"
+    ) -> Series[S1]: ...
+    def idxmin(self, axis: Axis | NoDefault = ..., skipna: bool = True) -> Series: ...
+    def idxmax(self, axis: Axis | NoDefault = ..., skipna: bool = True) -> Series: ...
+    def corr(
+        self,
+        other: Series,
+        method: CorrelationMethod = "pearson",
+        min_periods: int | None = None,
+    ) -> Series: ...
+    def cov(
+        self, other: Series, min_periods: int | None = None, ddof: int | None = 1
+    ) -> Series: ...
+    @property
+    def is_monotonic_increasing(self) -> Series[bool]: ...
+    @property
+    def is_monotonic_decreasing(self) -> Series[bool]: ...
     def hist(
         self,
-        by=...,
-        ax: PlotAxes | None = ...,
-        grid: bool = ...,
-        xlabelsize: int | None = ...,
-        xrot: float | None = ...,
-        ylabelsize: int | None = ...,
-        yrot: float | None = ...,
-        figsize: tuple[float, float] | None = ...,
-        bins: int | Sequence = ...,
-        backend: str | None = ...,
-        legend: bool = ...,
+        by: IndexLabel | None = None,
+        ax: PlotAxes | None = None,
+        grid: bool = True,
+        xlabelsize: float | None = None,
```

Comment:
Oh , the upstream docs of this, and other label size usages, will need to be updated to accept `float | str | None` (See the [matplotlib docs](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.tick_params.html)). I typed them as `int | None` here for now because they need to be fixed everywhere in pandas-stubs, not just this file. We can do that in a separate PR.

Reply:
created an issue https://github.com/pandas-dev/pandas-stubs/issues/852 and https://github.com/pandas-dev/pandas/issues/56876#issuecomment-1891041139